Lib/test/test_posix.py

```diff
@@ -724,11 +724,17 @@ def check_stat(uid, gid):
         chown_func(first_param, uid, -1)
         check_stat(uid, gid)
 
-        if uid == 0:
+        # On VxWorks root user id is 1 and 0 means no login user. And
+        # both are super users.
+        superuser_uids = ((0,) if sys.platform != "vxworks" else (0, 1))
+        if uid in superuser_uids:
```

Comment:
Code style suggestion:

```suggestion
        if sys.platform == "vxworks":
            # On VxWorks, root user id is 1 and 0 means no login user:
            # both are super users.
            is_root = (uid in (0, 1))
        else:
            is_root = (uid == 0)
        if is_root:
```